components/ColorPicker/src/CommunityToolkit.WinUI.Controls.ColorPicker.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="MSBuild.Sdk.Extras/3.0.23">
+  <PropertyGroup>
+    <ToolkitComponentName>ColorPicker</ToolkitComponentName>
+    <Description>This package contains ColorPicker.</Description>
+
+    <!-- Rns suffix is required for namespaces shared across projects. See https://github.com/CommunityToolkit/Labs-Windows/issues/152 -->
+    <RootNamespace>CommunityToolkit.WinUI.Controls.ColorPickerRns</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\Converters\src\CommunityToolkit.WinUI.Converters.csproj" />
+    <ProjectReference Include="..\..\Extensions\src\CommunityToolkit.WinUI.Extensions.csproj" />
+    <ProjectReference Include="..\..\Helpers\src\CommunityToolkit.WinUI.Helpers.csproj" />
```

Comment:
If we're just using the ColorHelper, wondering if we made a mistake and should have moved that to extensions. Too late now I guess. Just more dependencies than I would have expected.

Reply:
We should swap these to the absolute paths, as defined here -> https://github.com/CommunityToolkit/Windows/blob/main/Directory.Build.props#L10